janusgraph-dist/pom.xml

```diff
@@ -208,8 +208,10 @@
                         </goals>
                         <configuration>
                             <url>https://artifacts.elastic.co/downloads/elasticsearch/elasticsearch-${elasticsearch.version}-linux-x86_64.tar.gz</url>
-                            <outputFileName>elasticsearch.tar.gz</outputFileName>
-                            <outputDirectory>${project.build.directory}</outputDirectory>
+                            <outputFileName>elasticsearch-${elasticsearch.version}-linux-x86_64.tar.gz</outputFileName>
+                            <outputDirectory>${project.build.directory}/</outputDirectory>
+                            <unpack>true</unpack>
+                            <sha512>30764d5838009dc36d8f5c6c7249e65f323b5bd843027b47b37b91566d28cac95c4b5e6db1decce748f9ccd404a7ca4ba4fa9632bcd8b43fb27b0d93cc7b8f27</sha512>
```

Comment:
Is this the hash of the archive? Then we'll have to update it every time we update to another version, right? Do we want to do that for security reasons or do we maybe want to omit this check here?

Reply:
We have to update each time, but otherwise maven thought the file is already cached. So you run the tests with old version.

Reply:
OK, then I guess we have to update the values manually each time.